source/rainerscript/configuration_objects.rst

```diff
@@ -3,6 +3,25 @@ configuration objects
 
 Note: configuration object parameters are case-insensitive.
 
+All configuration objects have a parameter "config.enable".
+For auto-generated configs, it is useful to have the ability to disable some
+config constructs even though they may be specified inside the config. This
+can be done via the ``config.disable`` parameter.
```

Comment:
Both `config.disable` and `config.enable` parameters are available?

Reply:
no, I'll clear up. it's a boolean parameter so you have "on" and "off" values.